src/io.js

```diff
@@ -1,17 +1,17 @@
-
 import fs from 'fs';
-// import path from 'path';
 import colors from 'colors';
 
-export const outputPot = (filePath, contents, cb = () => {}) => {
+export const outputPot = (verbose, filePath, contents, cb = () => {}) => {
```

Comment:
I think `verbose` should be the last argument, both since it's the least important one, and as that would prevent breaking changes. Perhaps it could be wrapped in an `options` argument, giving room for more options later on.